### PR TITLE
Release 14.0.0

### PR DESCRIPTION
**Pull request by Mrtenz**
This is the release candidate for version `14.0.0`.

### CODE REVIEW

File: packages/typescript/package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
     "vitest": "^2.1.1"
   },
   "peerDependencies": {
-    "@metamask/eslint-config": "^13.0.0",
+    "@metamask/eslint-config": "workspace:^",
```
Comment (MajorLift) [inline packages/typescript/package.json:56]: Does this show up on npm as `workspace:^`, or as a specific version string? It looks like it's the latter for snaps packages? Does that work with no additional configuration?

Comment (Mrtenz) [reply packages/typescript/package.json:56]: Yarn replaces `workspace:^` with the actual version before publishing, so this works without any additional configuration.

Comment (MajorLift) [reply packages/typescript/package.json:56]: Nice! Good to know.

**Review by MajorLift — APPROVED** [review]
LGTM! Excited to see this go out.

**Pull request merged**